clients/privacy-center/features/common/property.slice.ts

```diff
@@ -0,0 +1,37 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+import { useAppSelector } from "~/app/hooks";
+import type { RootState } from "~/app/store";
+import { Property } from "~/types/api";
+
+interface PropertyState {
+  property?: Property;
+}
+const initialState: PropertyState = {};
+
+export const propertySlice = createSlice({
+  name: "property",
+  initialState,
+  reducers: {
+    /**
+     * Load property, replacing the current state entirely.
+     */
+    loadProperty(draftState, { payload }: PayloadAction<Property | undefined>) {
+      if (process.env.NODE_ENV === "development") {
+        // eslint-disable-next-line no-console
+        console.log("Loading Privacy Center property into Redux store...");
+      }
```

Comment:
It might be worth creating a `debugLog` util for privacy center like fides-js has. Not necessary before merge, just a suggestion for future reference.